src/aws-util/Cargo.toml

```diff
@@ -16,9 +16,16 @@ aws-smithy-client = { version = "0.36.0", default-features = false }
 aws-smithy-http = "0.36.0"
 aws-types = "0.6.0"
 mz-http-proxy = { path = "../http-proxy", features = ["hyper"] }
+openssl-sys = { version = "0.9.72", features = ["vendored"] }
```

Comment:
add back the original comment?
```suggestion
# Make sure the "vendored" feature makes it into the transitive dep graph of
# every aws user, so that we don't attempt to link against the system OpenSSL
# library.
openssl-sys = { version = "0.9.72", features = ["vendored"] }
```

Reply:
It's already down below!